src/EUTelProcessorGeometricClustering.cc

```diff
@@ -327,184 +328,177 @@ void EUTelProcessorGeometricClustering::geometricClustering(LCEvent * evt, LCCol
 		minX = minY = maxX = maxY = 0;
 		geoDescr->getPixelIndexRange( minX, maxX, minY, maxY );
 
```

Comment:
I guess most of this is just indentation? If possible don't change this as this makes comparing versions a pain in the ass!